src/views/HomeView.vue

```diff
@@ -153,11 +192,23 @@ export default class HomeView extends Vue {
       this.outputTextarea = "";
     }
   }
-
   async mounted() {
-    // await getPhones();
-    // await addProduct();
-    // await deleteProduct(1);
+    await getPhones();
+    await addProduct();
+    await deleteProduct(1);
+    this.checkLocalStorage("selectedLangs");
+    this.checkLocalStorage("translations");
+  }
+
+  @Watch("selectedLangs", { deep: true })
+  onLangsChanged(newValue: Record<TLangs, Languages>) {
+    localStorage.selectedLangs = JSON.stringify(newValue);
```

Comment:
it's not necessary to add a `watcher`. You can just update the localStorage in `changeLanguage` method. As far as I understand this is a single place where the selected languages change.